src/penn_chime/models.py

```diff
@@ -92,14 +92,20 @@ def __init__(self, p: Parameters):
                 self.i_day)
             min_loss = 2.0**99
             dts = np.linspace(1, 15, 29)
-            losses = np.zeros(dts.shape[0])
+            losses = np.full(dts.shape[0], np.inf)
             self.current_hospitalized = p.current_hospitalized
             for i, i_dt in enumerate(dts):
                 intrinsic_growth_rate = get_growth_rate(i_dt)
                 self.beta = get_beta(intrinsic_growth_rate, self.gamma, self.susceptible, 0.0)
                 self.beta_t = get_beta(intrinsic_growth_rate, self.gamma, self.susceptible, p.relative_contact_rate)
 
                 self.run_projection(p, self.gen_policy(p))
+
+                # Skip values the would put the fit past peak
+                peak_admits_day = self.admits_df.hospitalized.argmax()
```

Comment:
Perhaps store the unchanging argmax outside of the loop.

Reply:
That is changing - each call to `run_projections` generates a new `self.admits_df`

Reply:
Unless I misunderstood you?